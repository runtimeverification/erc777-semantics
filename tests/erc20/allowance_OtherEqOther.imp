test.preamble();

// Testing allowance(Owner,Spender) when Spender = Owner != caller

test.allowance(2, 2);
