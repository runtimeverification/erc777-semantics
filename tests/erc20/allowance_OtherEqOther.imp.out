Allowance of 2 for 2 is 0
